versions.nix

```diff
@@ -4,10 +4,30 @@
       portable = "12c58dcd8f61d18567fda0689ba64cb4b8c70a2d";
       openbsd = "dfa32b728c451dab0c97c854fbe23bc7b9be25ad";
     };
+    "9.2" = {
+      portable = "6d8627a9ef5606f4df11534e3b88fa52b3004bb1";
+      openbsd = "7d2c70c4b4060e915c367d1e9366a8c098cc1bc4";
+    };
     "9.1" = {
       portable = "a8aadf11866008666e17c65da76132659942fe2c";
       openbsd = "a59ae351b4f900cf90b01e755118290047fcdb28";
     };
+    "9.0" = {
+      portable = "47fb45fd13e402a21b51c794a76511a526b5d556";
+      openbsd = "d64b7617dc536b6c76a0c87113c4586479c4a3d2";
+    };
+    "8.9" = {
+      portable = "ee15321f92158f5b78ca7da3b33a7a7d26fcf91d";
```

Comment:
I see `70efa83594d4589521814fa782e1e453caf97697` as the portable hash for 8.9 here: https://github.com/rpki-client/rpki-client-portable/tags